connectors/src/connectors/google_drive/index.ts

```diff
@@ -132,31 +131,7 @@ export async function createGoogleDriveConnector(
     googleDriveConfigurationBlob
   );
 
-  try {
```

Comment:
Since we only register webhooks for Google Drives we are syncing, there is no need to do a global
webhook subscription anymore. We'll subscribe to the appropriates webhooks when the user selects the folders to sync.